src/gleam/http/request.gleam

```diff
@@ -67,7 +67,12 @@ pub fn from_uri(uri: Uri) -> Result(Request(String), Nil) {
 /// If the request does not have that header then `Error(Nil)` is returned.
 ///
 pub fn get_header(request: Request(body), key: String) -> Result(String, Nil) {
-  list.key_find(request.headers, string.lowercase(key))
+  use #(_, value) <- result.try(
+    list.find(request.headers, fn(header) {
+      string.lowercase(header.0) == string.lowercase(key)
```

Comment:
Let's add a var for the "lower-cased" key before using `list.find` to avoid lower-casing the key many times.

Reply:
Great suggestion